.github/workflows/publish-storybook.yml

```diff
@@ -0,0 +1,26 @@
+name: Build storybook & deploy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bucket: [dev-design.va.gov, staging-design.va.gov, design.va.gov]
+    steps:
+      - uses: actions/checkout@master
+      - name: Install dependencies
+        run: yarn install
+      - name: Build Storybook
+        run: yarn build-storybook
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: "us-gov-west-1"         
+      - name: S3 Sync Storybook build
+        run: |
+          aws s3 sync ./storybook-static s3://${{ matrix.bucket }}/storybook --acl public-read --follow-symlinks --delete
```

Comment:
It is intentional. This is partly due to the original action I used being based around `sync` rather than `cp`. However, as I was working on the action, I found [this SO answer](https://stackoverflow.com/a/64728207) which explains that `sync` results in less data being sent, which seems desirable.

Reply:
Awesome! I totally missed that this was `sync` and it was previously `cp`. Good call. :100: